openshift/ci-operator/knative-images/client/Dockerfile.cliartifacts

```diff
@@ -0,0 +1,19 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
+
+RUN yum -y install zip && yum clean all
+RUN bash package_cliartifacts.sh
+RUN mkdir -p /usr/share/kn/{linux_amd64,macos,windows} &&
+    mv kn-linux-amd64.tar.gz /usr/share/kn/linux_amd64/kn-linux-amd64.tar.gz && \
+    mv kn-macos-amd64.tar.gz /usr/share/kn/macos/kn-macos-amd64.tar.gz && \
+    mv kn-windows-amd64.zip /usr/share/kn/windows/kn-windows-amd64.zip
```

Comment:
The input to `package_cliartifacts.sh` (i.e. cross platform binaries, LICENSE and the script itself) are fed from CI.  
See https://github.com/openshift/release/pull/7404/files#diff-072f710a93e689d38ab54a2831351f1eR22